### PR TITLE
Make Introduction section and reflect IS-11 renaming

### PR DESCRIPTION
**Pull request by N-Nagorny**
…lect Sink Metadata Processing -> Flow Compatibility Management renaming

### CODE REVIEW

File: docs/EDID to Receiver Caps Mapping.md [hunk 1]
```diff
@@ -6,49 +6,34 @@
...
 
-_See also the [NMOS Glossary](https://specs.amwa.tv/nmos/branches/main/docs/4.0._Glossary.html), and definitions within RFCs._
+The proposed mapping is affected when the Receiver is:
 
-### Gateway Device
+- Capable of altering signal (e.g. it is able to consume 4K and downscale it before passing to the Output which downstream counterpart supports up to HD resolutions)
+- Constrained beyond the capabilities of the downstream counterpart of the Output (e.g. it supports resolution up to 1080p but the Output is connected to a 4K monitor)
 
-A device which encapsulates a digital video or/and audio signal (usually, from a physical connection) into IP packets. Conversely, the device MAY perform the reverse operation for IP packets to a digital signal.
+The Receiver SHALL include any types of conversion, omit any restrictions, or reflect the internal processing in own Receiver Capabilities.
```
Comment (alabou) [inline docs/EDID to Receiver Caps Mapping.md]: Could you explain what this means.

File: docs/EDID to Receiver Caps Mapping.md [hunk 2]
```diff
@@ -6,49 +6,34 @@
...
-- Using any internal processing before passing the signal to the Sink (e.g. support multiple Sinks per Receiver and use the greatest common denominator to constrain itself)
+## Use of Normative Language
 
-The Gateway Device SHALL include any types of conversion, omit any restrictions, or reflect the internal processing in the capabilities of the Receiver.
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119][RFC-2119].
 
 ## Video Receivers
 
-If an [IS-04][IS-04] Video Receiver is associated with a Sink which has EDID, its supported video formats MUST be mapped into Receiver's Capabilities according to the rules below.
+If an [IS-04][IS-04] Video Receiver is associated with an Output which has EDID, its supported video formats MUST be mapped into Receiver's Capabilities according to the rules below.
```
Comment (alabou) [inline docs/EDID to Receiver Caps Mapping.md]: If an [IS-04][IS-04] Video Receiver is associated with an Output which has an EDID, the optional mapping of the EDID supported video formats into Receiver's Capabilities SHALL be performed according to the rules below.

File: README.md [hunk 3]
```diff
@@ -11,11 +11,10 @@
 
 ## Why does it matter?
 
-- Receivers associated with physical devices providing EDID need to present their capabilities to Controllers
+- Receivers associated with Outputs that provide EDID need to present its contents to Controllers
```
Comment (alabou) [inline README.md]: Receivers associated with Outputs that provide an EDID may need to present its contents to Controllers

Comment (N-Nagorny) [reply README.md]:
```suggestion
- Receivers associated with Outputs that provide EDID may need to present its contents to Controllers
```

File: docs/EDID to Receiver Caps Mapping.md (hunk 2 above)
Comment (N-Nagorny) [reply docs/EDID to Receiver Caps Mapping.md]:
```suggestion
If an [IS-04][IS-04] Video Receiver is associated with an Output which has an EDID, the optional mapping of the EDID supported video formats into Receiver's Capabilities SHALL be performed according to the rules below.
```

File: docs/EDID to Receiver Caps Mapping.md (hunk 1 above)
Comment (N-Nagorny) [reply docs/EDID to Receiver Caps Mapping.md]: It is "SHALL" for circumstances described in the previous paragraph.
```
The proposed mapping is affected when the Receiver is:

- Capable of altering signal (e.g. it is able to consume 4K and downscale it before passing to the Output which downstream counterpart supports up to HD resolutions)
- Constrained beyond the capabilities of the downstream counterpart of the Output (e.g. it supports resolution up to 1080p but the Output is connected to a 4K monitor)
```

Maybe it makes sense to replace these two paragraphs with
```
The proposed mapping SHALL be affected when the Receiver is:

- Capable of altering signal (e.g. it is able to consume 4K and downscale it before passing to the Output which downstream counterpart supports up to HD resolutions)
- Constrained beyond the capabilities of the downstream counterpart of the Output (e.g. it supports resolution up to 1080p but the Output is connected to a 4K monitor)
```

Comment (alabou) [reply docs/EDID to Receiver Caps Mapping.md]: Why not say it that way "The proposed mapping provides the mechanism for converting EDID information into Receiver Capabilities. The effective Receiver Capabilities MAY differ from those obtained from the proposed mapping when the Receiver is:
- Capable of altering signal (e.g. it is able to consume 4K and downscale it before passing to the Output which downstream counterpart supports up to HD resolutions) 
- Constrained beyond the capabilities of the downstream counterpart of the Output (e.g. it supports resolution up to 1080p but the Output is connected to a 4K monitor)"

Comment (N-Nagorny) [reply docs/EDID to Receiver Caps Mapping.md]: Thanks, that's great language.

**Review by alabou — APPROVED** [review]

**Pull request merged**